cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNetworkConnector.java

```diff
@@ -126,6 +133,19 @@ public String getNetworkCidr(Network network, CloudCredential credential) {
         return vpcCidrs.get(0);
     }
 
+    @Override
+    public List<CloudSubnet> selectSubnets(List<CloudSubnet> subnetMetas, Tunnel tunnel, boolean ha, boolean preferPrivateAnyways) {
```

Comment:
I think an object would better here which contains all of the property

Reply:
Yes, definitely, I thought the same. Thanks.